models/buoys/buoys.sdf

```diff
@@ -113,15 +101,6 @@
           <diffuse>0.3 0.3 0.3 1</diffuse>
         </material>
         </visual>
-        <collision name='red_rod_collide'>
```

Comment:
Why have you removed the collision of these? I would have thought that it would be useful to have as the buoys are actually anchored and we don't want the sub getting caught up in them.